src/screens/EncounterHomeScreen.tsx

```diff
@@ -146,6 +177,13 @@ export const EncounterHomeScreen = ({ navigation, route }: any) => {
               }}
               title={texts.encounter.myData}
             />
+            {encounterCategoryId && (
+              <Button invert title={texts.encounter.toCategory} onPress={onPressToCategory} />
+            )}
+            {additionalInfo?.length && (
```

Comment:
do we need `!!` cast to booleans  here in some cases for the conditions to not make react native think to render something? is this still a thing or maybe unnecessary since some newer version?

Reply:
it should not be needed unless someone specifically adds an empty string into the setting.

i added it to both line 181 and 183, just to avoid having that possibility.